x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -511,12 +511,24 @@ export const getSavedObjectTypes = (
                   id: { type: 'keyword' },
                 },
               },
+              kibana_api_key: {
+                dynamic: false,
+                properties: {
+                  id: { type: 'keyword' },
+                },
+              },
             },
           },
           preset: {
             type: 'keyword',
             index: false,
           },
+          sync_integrations: { type: 'boolean' },
```

Comment:
I believe we may not need those property here, if we remove dynamic mappings, I have done this for package policy before https://github.com/juliaElastic/kibana/commit/721b4beb1ae7f12171a34d50ba6068b8c2d7288e#diff-4e164e3802d5171bf96a2cf9c91c20e97c5e0b74b2f93187a072d9a3139f1c18 
it's weird as it need an empty migration but seems to work

Reply:
Great, it looks like we can simplify like this, I'll remove the new props